src/locales/es/interactives/global-climate/global-climate-1.json

```diff
@@ -311,5 +311,7 @@
       "showHeat"
     ],
     "perTick": []
+    "lang": "es",
+  "i18nMetadata": "locales/metadata/interactives/global-climate/global-climate-1.json"
```

Comment:
This is correct. We should also add exactly the same property and value to the original interactive (`interactives/global-climate/global-climate-1.json`), so it knows that the Spanish version exists.